Firestore/core/src/api/aggregate_query.cc

```diff
@@ -42,11 +43,28 @@ AggregateQuery::AggregateQuery(Query query,
     : query_{std::move(query)}, aggregates_{std::move(aggregates)} {
 }
 
-void AggregateQuery::Get(AggregateQueryCallback&& callback) {
+void AggregateQuery::GetAggregate(AggregateQueryCallback&& callback) {
   query_.firestore()->client()->RunAggregateQuery(query_.query(), aggregates_,
                                                   std::move(callback));
 }
 
+// TODO(b/280805906) Remove this count specific API after the c++ SDK migrates
+// to the new Aggregate API
+void AggregateQuery::Get(CountQueryCallback&& callback) {
+  this->GetAggregate([callback](const StatusOr<ObjectValue>& result) {
+    if (result.ok()) {
+      auto count_value =
```

Comment:
Also, here, please replace `auto` with the actual type, to improve readability.

Reply:
updated this, but I do feel like using `auto` is easier to read. The type here is pretty cumbersome to read.